### PR TITLE
Expand Nightmare Snippets

### PR DESCRIPTION
**Pull request by TheMurderUnicorn**
#### Summary
Content "Add More Nightmares"

#### Purpose of change
There were only like 50 nightmares, so duplicates were common and it seemed like an area of the game I could expand without messing anything up.  Submitted for @I-am-Erk 's approval mainly.

#### Describe the solution
I wrote a bunch of nightmares.  Honestly, I forgot they were supposed to be generic until I was submitting this just now, so... a bunch of them might end up getting the axe, I don't know.  I also chronically overuse commas and never write in present tense so I might have slipped into past tense or something, idk I'm a shit editor. 

#### Describe alternatives you've considered
Being less of a gothy emo guy, which definitely affects my world view and nightmares o_o

#### Testing
Just made sure they booted up in the game, had the right typographical `"`, made sure the `\"` were working properly, etc.

#### Additional context
I have no emotional attachment to any of these, hack away the ones you hate and I'll just mash approve on any changes you want to make.  If they WERE to all be added (which they probably won't) this would more than double the number of existing nightmares.

And if I've done something staggeringly wrong and they all need rejected, please let me know what I did wrong and I will try again in the future.

### CODE REVIEW

File: data/json/snippets/nightmares.json [hunk 1]
```diff
@@ -53,7 +53,73 @@
...
+      "You wake up with a feeling that you’ve forgotten something important.  It takes you several seconds to remember where you are.",
+      "You wake up feeling lost and alone, thinking of someone you used to know.",
+      "You awaken from a nightmare but remember none of it.  Your heart races and your breathing is unsteady.",
+      "You dream of coming down the stairs to find an old friend in a hallway, their back to you.  When you call out to them, their face slowly turns in your direction, but the dream melts away.",
+      "You awaken from a nightmare in which you are lying on a butcher’s table, surrounded by eager undead, metal blades shining in their rotten hands.",
+      "You awaken from a nightmare about being chased, your companion leading you through dark corridors to escape.",
+      "You dream of being dirty, the stench of rotting flesh wafting from your clothing, from your mouth, from your pores.",
+      "Your sleeping mind wanders.  There’s a man, and in the dream you’re best friends.  His eyes level on yours as you offer a hasty farewell. \"Don’t ever stop running,\" he whispers.",
```
Comment (DSeyka) [inline data/json/snippets/nightmares.json:74]:
```suggestion
      "Your sleeping mind wanders.  There’s a man, and in the dream you’re best friends.  His eyes level with yours as you offer a hasty farewell. \"Don’t ever stop running,\" he whispers.",
```

File: data/json/snippets/nightmares.json [hunk 2]
```diff
@@ -53,7 +53,73 @@
...
+      "You dream you are a slug, and all is well until a child stands over you with a salt shaker.",
+      "You dream you’re walking in the woods.  There’s no one around and your phone is dead.  That’s when you spot him.",
+      "You wake from a nightmare of being slowly devoured, starting with your toes.",
+      "You wake from a nightmare of being torn to pieces, lying in the filth of the road and waiting for the end.",
+      "A seemingly endless nightmare plagues you.  Years pass, and you are always struggling alone in an unforgiving world, dying in solitude of old age.",
+      "You dream that you are the last person on Earth.  You’re surrounded by \"people\" but you know they aren’t human anymore.",
+      "You wake from a nightmare in which you endlessly stacked boxes in a factory, overseen by an inhuman figure clothed in shadow.",
+      "You dream of a behemoth, towering over you and covered in bony tissue, its face twisted with rage.  You run for as long as you can, but when its massive fist comes down, you wake.",
+      "You dream of a beautiful and regal unicorn.  You feed it gems as it nuzzles your hand.  When you’ve run out of stones, it kicks you to death.",
```
Comment (DSeyka) [inline data/json/snippets/nightmares.json:117]:
```suggestion
      "You dream of a beautiful and regal unicorn.  You feed it gems as it nuzzles your hand.  When you run out of stones, it kicks you to death.",
```

File: data/json/snippets/nightmares.json [hunk 3]
```diff
@@ -53,7 +53,73 @@
...
+      "You wake from a nightmare in which it had all started over again, 911 ringing busy, the fires, and death in every street.",
+      "You dream your boss is scolding you for allowing a zombie through security.",
```
Comment (DSeyka) [inline data/json/snippets/nightmares.json:119]:
```suggestion
      "You dream that your boss is scolding you for allowing a zombie through security.",
```

**Comment by DSeyka** [comment]
All my suggestions are more about style than grammar and can be disregarded if deemed unnecessary.

**Comment by Qrox** [comment]
You should replace `‘’` in the text with `'`.

File: data/json/snippets/nightmares.json (hunk 3 above)
Comment (TheMurderUnicorn) [reply data/json/snippets/nightmares.json:119]: I prefer the original for this one, I don't know if it matters

**Comment by TheMurderUnicorn** [comment]
> You should replace `‘’` in the text with `'`.

Why?  Single quotes are for quotes within dialogue, and I know the game uses double quotes in various newspaper snippets.

**Pull request merged**